src/components/form-tags/form-tags.spec.js

```diff
@@ -348,17 +348,17 @@ describe('form-tags', () => {
     expect(wrapper.find('.invalid-feedback').exists()).toBe(false)
     expect(wrapper.find('.form-text').exists()).toBe(false)
 
-    $input.element.value = 'thre'
+    $input.element.value = 'three'
```

Comment:
This was actually needed as part of the test.  `thre` passes, but `three` doesn't pass validation.

Reply:
Updated the tests to better emulate individual characters being typed.